ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -49,7 +46,7 @@ const SecretLabelPrefix = "ecs-local.secret"
 
 // ConvertToDockerCompose creates the payload from an ECS Task Definition to be written as a docker compose file
 func ConvertToDockerCompose(taskDefinition *ecs.TaskDefinition) ([]byte, error) {
-	services := []composeV3.ServiceConfig{}
+	var services []composeV3.ServiceConfig
```

Comment:
Is instantiating this with a `var` (e.g. nil slice) because we want the JSON marshalling to return nil versus an empty slice (in line 68)? (See: https://golang.org/pkg/encoding/json/#Marshal)

Reply:
Sure. Ultimately, we should have a check that there is at least one service defined anyway; whether its pre-marshalled form is nil or empty shouldn't matter that much i don't think.